### DISCUSSION TITLE
Where did my testnet Filecoin go? (Sept 7th upgrade)

### DISCUSSION THREAD

**Discussion by coffee115599** [filecoin-project/lotus#3652] [General Q&A] [answered]
**Describe the bug**
The FILs in my wallet just gone....

**To Reproduce**
lotus-miner actor control list:
```
2020-09-08T08:21:09.269Z        WARN    storageminer    storage/addresses.go:63 control address didn't have enough funds for PoSt message       {"address": "t019035", "required": "1 FIL", "balance": "0 FIL"}
name       ID       key           use         balance
owner      t015051  t3vj7frno...  other post  7.801723210885638967 FIL
worker     t015051  t3vj7frno...  other post  7.801723210885638967 FIL
control-0  t019035  t3r47ytey...              0 FIL
```

**Expected behavior**
A few hours ago
t015051 had 500+ FILs
t019035 had Some FILs
t015233 had 100+ FILs
But now, all gone..........
How can I continue then ?

**Screenshots**

**Version (run `lotus version`):**
Daemon:  0.6.1+git.4045ffea+api0.14.0
Local: lotus version 0.6.1+git.4045ffea

**Additional context**

**Comment by ccpmaxi** [comment]
me too. The balance of wallet is zero now.

**Comment by coffee115599** [comment]
https://spacerace.faucet.glif.io/
Always return "Request failed with status code 500"
How can I continue then ?

**Comment by xstc-cc** [comment]
me too  ,   t1ewxfvzjxcmk44zxzgiexatdiviqerfmrqficgji

**Reply by xiaozhucdj** [reply]
me too     f1gwtbiblk34jt3areyhkwgvjeuohvp57usfe5egi

**Comment by momack2** [comment, marked as answer]
Please see these two threads in Filecoin Slack: 
- https://filecoinproject.slack.com/archives/C019UFEACBT/p1599474876263300
- https://filecoinproject.slack.com/archives/C0179RNEMU4/p1599546507186600

Space Race FIL reallocation is intentional to rebalance the base fee and expectations for FIL availability.
